lib/rubocop/cop/lint/assignment_in_condition.rb

```diff
@@ -24,7 +24,10 @@ module Lint
       class AssignmentInCondition < Cop
         include SafeAssignment
 
-        MSG = 'Assignment in condition - you probably meant to use `==`.'.freeze
+        MSG = [
```

Comment:
Ah! Sorry, I didn't realize. OK, I just amended it to handle both scenarios. Let me know if I didn't do that right! 😃